docs/pages/repo/docs/core-concepts/caching.mdx

```diff
@@ -44,84 +42,23 @@ Let's say that you run the task again without changing any of its inputs:
-
-When you run `turbo run build test`, Turborepo will execute your build and test scripts,
-and cache their `output`s in `./node_modules/.cache/turbo`.
-
-<Callout type="info">
-  Pro Tip for caching ESLint: You can get a cacheable pretty terminal output
-  (even for non-errors) by setting `TIMING=1` variable before `eslint`. Learn
-  more over in the [ESLint
-  docs](https://eslint.org/docs/latest/developer-guide/working-with-rules#per-rule-performance).
-</Callout>
-
-## Configuring Cache Inputs
-
-A workspace is considered to have been updated when any of the files in that workspace have changed.
-However, for some tasks, we only want to rerun that task when relevant files have changed.
-Specifying `inputs` lets us define which files are relevant for a particular task. For example, the
-`test` configuration below declares that the `test` task only needs to execute if a `.tsx` or `.ts` file
-in the `src/` and `test/` subdirectories has changed since the last execution.
-
-```jsonc
-{
-  "$schema": "https://turbo.build/schema.json",
-  "pipeline": {
-    // ... omitted for brevity
-
-    "test": {
-      // A workspace's `test` task depends on that workspace's
-      // own `build` task being completed first.
-      "dependsOn": ["build"],
-      // A workspace's `test` task should only be rerun when
-      // either a `.tsx` or `.ts` file has changed.
-      "inputs": ["src/**/*.tsx", "src/**/*.ts", "test/**/*.ts"]
-    }
-  }
-}
-```
+Restoring files and logs from the cache happens near-instantaneously. This can reduce your build times from minutes or hours down to seconds or milliseconds. Although specific results will vary depending on the shape and granularity of your codebase's dependency graph, most teams find that they can reduce their overall monthly build time by around 40-85% with Turborepo's caching.
 
 ## Turn off caching
 
-Sometimes you really don't want to write the cache output (e.g. when you're using [`next dev`](https://nextjs.org/docs/api-reference/cli#development) or `react-scripts start` for live reloading). To disable cache writes, append `--no-cache` to any command:
+In some environments you don't want to write the cache output. To disable cache writes, append `--no-cache` to any command. This will run `dev` (and all tasks that it dependsOn) in all workspaces, but it won't cache the output.
 
 ```shell
-# Run `dev` npm script in all workspaces in parallel,
-# but don't cache the output
 turbo run dev --no-cache
 ```
 
 Note that `--no-cache` disables cache writes but does not disable cache reads. If you want to disable cache reads, use the `--force` flag.
 
-You can also disable caching on specific tasks by setting the [`pipeline.<task>.cache`](/repo/docs/reference/configuration#cache) configuration to `false`:
+You can also permanently disable caching on specific tasks by setting the [`pipeline.<task>.cache`](/repo/docs/reference/configuration#cache) configuration to `false`:
```

Comment:
I don't know that "permanently" belongs here. I mean, yes, for some definitions of permanent, but if I remove the config setting, it's no longer disabled, which is not permanent. 

Maybe there's another way to phrase it? "You can configure specific tasks to skip writing to the cache by ..." or something like that?

Reply:
persistently?